### PR TITLE
Feat/163223759/Deploy app to heroku

### PR DESCRIPTION
**Pull request by XROLE**
#### What does this PR do?
- Deploy authors-haven to Heroku

#### Description of Task to be completed?
- [x] Add build script to build out all files 
- [x] Move babel files from devdependencies to dependencies
- [x] Refactored start script to use build directory
- [x] Add script to clean the build file before a new build
- [x] Deploy app to Heroku 

#### How should this be manually tested?
- Visit [authors-haaven](https://authors-haaven.herokuapp.com/)

#### What are the relevant pivotal tracker stories?
[#163223759]

### CODE REVIEW

**Comment by oreoluwade** [comment]
You should add the `build` directory to `.gitignore`. The build will happen on heroku. Also, delete the build directory itself. It will not be merged to develop.

**Comment by XROLE** [comment]
> You should add the `build` directory to `.gitignore`. The build will happen on heroku. Also, delete the build directory itself. It will not be merged to develop.

Ok. Doing that right away. I noticed the hound violation late. I will handle that as well. Thanks

File: package.json [hunk 1]
```diff
@@ -16,8 +18,13 @@
   "author": "Andela Simulations Programme",
   "license": "MIT",
   "dependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.3",
     "babel-plugin-transform-runtime": "^6.23.0",
     "babel-polyfill": "^6.26.0",
+    "babel-preset-env": "^1.7.0",
+    "babel-register": "^6.26.0",
```
Comment (oreoluwade) [inline package.json]: Was this dependency used anywhere? If no, it will need to be removed.

File: config/config.js [hunk 2]
```diff
@@ -12,7 +12,8 @@ module.exports = {
     operatorsAliases: false
   },
   production: {
-    url: process.env.PRODUCTION_DATABASE_URL,
+    use_env_variable: 'DATABASE_URL',
```
Comment (oreoluwade) [inline config/config.js]: Is the `use_env_variable` variable used somewhere?

File: package.json (hunk 1 above)
Comment (XROLE) [reply package.json]: Babel-preset was used but babe-register was not. Removing babel-register asap

File: config/config.js (hunk 2 above)
Comment (XROLE) [reply config/config.js]: Yes, I saw it as part of settings I need to do to allow heroku run sequelize migrations on heroku postgress.

**Review by oreoluwade — APPROVED** [review]

**Pull request merged**